gridstatus/ercot.py

```diff
@@ -533,16 +548,12 @@ def _get_todays_outlook_non_forecast(self, date, verbose=False):
             date,
             self.default_timezone,
         ), "Only today's data is supported"
-        url = self.BASE + "/supply-demand.json"
 
-        msg = f"Fetching {url}"
-        log(msg, verbose)
+        supply_demand_json = self._get_supply_demand_json(verbose=verbose)
 
-        r = self._get_json(url)
+        date = self._get_date_from_supply_demand_json(supply_demand_json)
```

Comment:
i dont think date is actually used in this function? not sure why this line was here in the first place

so, we can also get rid of `_get_date_from_supply_demand_json`